src/main/java/org/graylog2/log4j2/GelfAppender.java

```diff
@@ -141,6 +149,40 @@ public void append(LogEvent event) {
         }
     }
 
+    private Formatter buildStackTrace(Formatter formatter, Throwable throwable) {
+
+        for (StackTraceElement stackTraceElement : throwable.getStackTrace()) {
+            formatter.format("%s.%s(%s:%d)%n",
+                    stackTraceElement.getClassName(),
+                    stackTraceElement.getMethodName(),
+                    stackTraceElement.getFileName(),
+                    stackTraceElement.getLineNumber());
+        }
+        Throwable nextThrowable = throwable.getCause();
+        if (nextThrowable != null) {
+            formatter.format("%n---> Caused by: %s%n", nextThrowable.getMessage());
```

Comment:
I added the tacky `--->` here on purpose, as it's otherwise extremely difficult to parse the stacktrace in Graylog, i.e., as Graylog flattens the stacktrace, a visual marker is necessary to identify the causes